src/builtins/core/duration/normalized.rs

```diff
@@ -686,65 +766,74 @@ impl InternalDurationRecord {
             end_duration,
         })
     }
-    // TODO: Add assertion into impl.
-    // TODO: Add unit tests specifically for nudge_calendar_unit if possible.
-    fn nudge_calendar_unit(
+
+    fn nudge_calendar_unit_total(
```

Comment:
nit: mention what spec item this is and why it is different

question: why is this different?